vega/core/scheduler/worker_env.py

```diff
@@ -137,7 +137,7 @@ def transition(self, key, start, finish, *args, **kwargs):
                     p.wait()
                     fh.close()
                     sub_pids = p.stdout.read().split()
-                    for spid in sub_pids:
+                    for spid in sub_pids[1:]:
```

Comment:
What are the failures caused by the current code?

Reply:
```
ERROR:root:Worker Plugin Error.
ERROR:root:Traceback (most recent call last):
  File "/vega/vega/core/scheduler/worker_env.py", line 142, in transition
    cuda_pid_set.add(int(spid))
ValueError: invalid literal for int() with base 10: b'kernel'
```
Because here `subpids` is `[b'kernel', b'596', b'745', b'746', b'747', b'748', b'749', b'750', b'751', b'752', b'753', b'754']`.

You can reproduce this error by running `data_augmentation/pba/pba.yml`